Libraries/Components/Navigation/NavigatorIOS.ios.js

```diff
@@ -270,6 +270,12 @@ var NavigatorIOS = React.createClass({
        */
       translucent: PropTypes.bool,
 
+      /**
+       * A Boolean value that indicates whether the interactive pop gesture is enabled. Useful
+       * for enabling/disabling the back swipe navigation gesture.
+       */
+      interactivePopGestureEnabled: PropTypes.bool,
```

Comment:
It's not a property on the route is it? Just on `<NavigatorIOS>`, so we should only have one propTypes definition.